app/react/Attachments/components/FileList.tsx

```diff
@@ -39,8 +40,16 @@ export class FileList extends Component<FileListProps> {
   }
 
   render() {
-    const { files } = this.props;
-
+    const { files, entity } = this.props;
+    const fileIndex = files.findIndex(file => {
```

Comment:
Please abstract this code to a different method

Reply:
On it

Reply:
I am done refactoring